backend/src/main/java/com/github/devbk84/backend/models/Order.java

```diff
@@ -0,0 +1,13 @@
+package com.github.devbk84.backend.models;
+
+import java.util.List;
+
+public record Order(
+        String id,
+        String payment,
+        List<Product> products,
+        String ordertBy
```

Comment:
This is supposed to reference the customer who placed the Order I guess?
Perhaps you could think if there's another name for this attribute

Reply:
(see if you could find a name that's a little more precise)